.doc_gen/metadata/aurora_Get-RDSOrderableDBInstanceOption_metadata.yaml

```diff
@@ -0,0 +1,23 @@
+aurora_DescribeOrderableDBInstanceOptions:
+  title: Describe options for &AURlong; DB instances using an &AWS; SDK
+  title_abbrev: Describe options for &AUR; DB instances
+  synopsis: describe options for &AUR; DB instances.
+  category:
+  languages:
+    PowerShell:
+      versions:
+        - sdk_version: 4
+          excerpts:
```

Comment:
Please add this property:
```
sdk_action: Get-RDSOrderableDBInstanceOption
```

This is late breaking requirement emerging from https://github.com/aws/aws-tools-for-powershell/pull/346